types/pannellum/pannellum-tests.ts

```diff
@@ -0,0 +1,35 @@
+import ".";
```

Comment:
import package here (like when consuming in real code)

Reply:
or skip imports, if this is a global script

Reply:
> import package here (like when consuming in real code)
> or skip imports, if this is a global script

Not sure if I understand. When using pannellum, we just import the library and it sets global object `window.pannellum` - the lib doesn't export anything.

https://github.com/mpetroff/pannellum/blob/2.5.6/src/js/pannellum.js#L24

Reply:
agree, but here one should import the module under test same way as expected when released, so tests cannot use file based imports. Use module name in the import, or skip import, this is global script, e.g.:
```suggestion
```